uint8 mode 
float32 scan_range
float32 scan_cycle_time
float32 error

byte FREE = 0
byte SCAN = 1
byte TRACK = 2